assets/src/js/search.js

```diff
@@ -1,137 +1,165 @@
-  $( '.activefilter-tag' ).off( 'click' ).on( 'click', function() {
-    $( '.p4-custom-control-input[value=' + $( this ).data( 'id' ) + ']' ).prop('checked', false );
-    $search_form.submit();
+  const activeFilterTags = [...document.querySelectorAll('.activefilter-tag')];
+  activeFilterTags.forEach(activeFilterTag => {
+    const filterId = activeFilterTag.dataset.id;
+
+    activeFilterTag.onclick = () => {
+      addSelectedFiltersToForm(false, filterId);
+      searchForm.submit();
+    };
   });
 
   // Clear all selected filters.
-  $( '.clearall' ).off( 'click' ).on( 'click', () => {
-    $( 'input[name^="f["]' ).prop( 'checked', false );
-    $search_form.submit();
-  });
+  const clearAllButton = document.querySelector('.clearall');
+  if (clearAllButton) {
+    clearAllButton.onclick = () => {
+      const selectedFilters = [...document.querySelectorAll('input[name^="f["]:checked')];
+      selectedFilters.forEach(selectedFilter => {
+        selectedFilter.checked = false;
+      });
+      searchForm.submit();
+    };
+  }
 
   // Add click event for load more button in blocks.
-  $load_more_button.off( 'click' ).on( 'click', function() {
-    const total_posts    = $(this).data('total_posts');
-    const posts_per_load = $(this).data('posts_per_load');
-    const next_page      = $(this).data( 'current_page' ) + 1;
-    $(this).data( 'current_page', next_page );
-
-    $.ajax({
-      url: localizations.ajaxurl,
-      type: 'GET',
-      data: {
-        action:          'get_paged_posts',
-        'search-action': 'get_paged_posts',
-        'search_query':  $( '#search_input' ).val().trim(),
-        'paged':         next_page,
-        'orderby': $( '#orderby', $search_form ).val(),
-        'query-string':  decodeURIComponent( location.search ).substr( 1 ) // Ignore the ? in the search url (first char).
-      },
-      dataType: 'html'
-    }).done(( response ) => {
-      // Append the response at the bottom of the results and then show it.
-      $( '.multiple-search-result .list-unstyled' ).append( response );
-      $( '.row-hidden:last' ).removeClass( 'row-hidden' ).show( 'fast' );
-
-      if (posts_per_load * next_page > total_posts) {
-        $load_more_button.hide();
-      }
-    }).fail(( jqXHR, textStatus, errorThrown ) => {
-      console.log(errorThrown); //eslint-disable-line no-console
-    });
-  });
+  const navSearchInput = document.getElementById('search_input');
+  const loadMoreButton = document.querySelector('.btn-load-more-click-scroll');
+  if (loadMoreButton) {
+    loadMoreButton.onclick = () => {
+      const { total_posts, posts_per_load, current_page } = loadMoreButton.dataset;
+      const nextPage = parseInt(current_page) + 1;
+      loadMoreButton.dataset.current_page = nextPage;
+
+      $.ajax({
+        url: localizations.ajaxurl,
+        type: 'GET',
+        data: {
+          action: 'get_paged_posts',
+          'search-action': 'get_paged_posts',
```

Comment:
We ended up [not changing](https://github.com/greenpeace/planet4-master-theme/pull/1560) the way requests are made, so you could also update this block code. When you rebase you'll have a conflict here, it's only because of this parameter that I removed.